runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -674,6 +681,58 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		stringStack[stringStackSize - 1] += titleBuilder.toString();
 	}
 
+	private void setLimitReset(int itemId)
+	{
+		String lastLimit = getLimitReset(itemId);
+		if (!lastLimit.equals("None"))
+		{
+			return; // do not update if previous limit has not expired
+		}
+		String itemIdStr = Integer.toString(itemId);
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+		LocalDateTime now = LocalDateTime.now();
+		String currentDateTime = now.format(formatter);
+		configManager.setConfiguration("gelimitreset." + client.getUsername().toLowerCase(),
```

Comment:
This should probably be group `grandexchange` key `limit.id.username` or something.

Reply:
I copied it from the setOffer method in the same file:
`private void setOffer(int slot, SavedOffer offer)
	{
		configManager.setConfiguration("geoffer." + client.getUsername().toLowerCase(), Integer.toString(slot), GSON.toJson(offer));
	}`

Changing gelimitreset to better fit the existing norms

Reply:
`configManager.setConfiguration("gelimitreset",
			client.getUsername().toLowerCase(), itemIdStr + currentDateTime);`